deploy/kubernetes/manifests/weavescope.yaml

```diff
@@ -0,0 +1,79 @@
+apiVersion: v1
```

Comment:
Why do we need this file here? The docs reference launcher URL, which is what we should do.